python-agent-tools/add-item-to-list/tool.py

```diff
@@ -63,9 +63,21 @@ def invoke(self, input, trace):
             write_from_dict=True
         )
         row = input.get("input", {})
+
+        # Log inputs and config to trace
+        trace.span["name"] = "WRITE_TO_SHAREPOINT_LIST_TOOL_CALL"
+        trace.inputs["list"] = self.list
+        trace.inputs["row"] = row
+        trace.attributes["config"] = self.config
```

Comment:
Won't work, `self.config` is not assigned in the `set_config`method

Reply:
Fixed in [711d7fb](https://github.com/dataiku/dss-plugin-sharepoint-tools/pull/2/commits/711d7fb955ab9b87d6e7d43a02566e7c9a84372c)